DEPLOYING.md

```diff
@@ -125,3 +174,62 @@ To check the logs for a specific service:
 sudo docker-compose -f docker-compose.yaml -f .docker-compose-db.yaml -f .docker-compose.vital-strategies-theme.yaml logs -f ckan
 ```  
 *(exit the logs by pressing Ctrl+C at any time)*
+
+To get inside a running container
+
+```
+sudo docker-compose -f docker-compose.yaml -f .docker-compose-db.yaml -f .docker-compose.vital-strategies-theme.yaml exec {service-name} bash
+```  
+
+## Creating the sysadmin user
+
+In order to create organizations and give other user proper permissions, you will need sysamin user(s) who has all the privileges. You can add as many sysadmin as you want. To create sysamin user:
+
+```
+# ssh into ckan container
+sudo docker-compose -f docker-compose.yaml -f .docker-compose-db.yaml -f .docker-compose.vital-strategies-theme.yaml exec ckan bash
+
+# Creat sysadmin user using paster CLI tool
+/usr/local/bin/ckan-paster --plugin=ckan sysadmin add {username} --password={password} --email={email} - /etc/ckan/production.ini
```

Comment:
question, does it work if you join them in one command?

```
sudo docker-compose -f docker-compose.yaml -f .docker-compose-db.yaml -f .docker-compose.vital-strategies-theme.yaml exec ckan \
/usr/local/bin/ckan-paster --plugin=ckan sysadmin add {username} --password={password} --email={email} - /etc/ckan/production.ini
```

Reply:
Hm worth to try